ui/src/routes.tsx

```diff
@@ -63,6 +63,7 @@ export default memo(function AppRoutes() {
   const isUserEmpty = R.either(R.isEmpty, R.isNil)(user);
 
   return isLoading ? (
+    // The global full page loading state is causing issues with infinite scroll. Remove this in future
```

Comment:
What issues is this causing?

Reply:
This is a global loader which is coming from `useFrontier` is displayed on the full screen. Ideally, for infinite scroll, the loader is only shown on the bottom of the table and not full screen.